install/helm/agones/templates/mutatingwebhook.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
We also need the [ValidatingWebhookConfiguration](https://github.com/GoogleCloudPlatform/agones/blob/master/install.yaml#L198)

Reply:
Actually, we don't right now. I lie - we can do this when we integrate into the build tooling.